src/scripts/modules/workspace/popovers/new/modals/new-media.coffee

```diff
@@ -26,6 +26,7 @@ define (require) ->
       else
         content = new Page
           title: options.title
+          content: '<p>Enter some content here. Format content, and drag and drop elements from the toolbar.</p>'
```

Comment:
Hmm, this makes me think we may have more hidden regressions because of this.  Some places may be calling `toJSON` and others may be getting it directly with `get('contents')`.

The default text should only be loading and visible after the page has loaded and been overwritten by the AJAX response, so what we're doing here may be more of a hack for the symptom instead of the problem.

I think this may be closely related to the issue of the progress bar not showing up when loading a page either.  There's probably a logic error in the body template to decide whether or not to show the content.  The content shouldn't be shown at all until the page is loaded, whereas right now it's showing the content (whether the default text or empty content).